### PR TITLE
Gem Pack - Add "trust any certificate" option

### PR DESCRIPTION
**Pull request by liormgem**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Adding the option to trust any certificate.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by liormgem** [comment]
> @liormgem - Please update the integration README to add this new configuration to the doc. Use:
> 
> ```shell
> demisto-sdk generate-docs --input Packs/Gem/Integrations/Gem/Gem.py
> ```
> 
> Also, please create a release notes entry for this change as well:
> 
> ```shell
> demisto-sdk update-release-notes --input Packs/Gem/Integrations/Gem/Gem.py --update-type revision
> ```

Done

File: Packs/Gem/Integrations/Gem/Gem.yml [hunk 1]
```diff
@@ -41,6 +41,11 @@ configuration:
     type: 8
     section: Connect
     required: false
+  - display: Trust any certificate (not secure)
+    name: unverify
```
Comment (RotemAmit) [inline Packs/Gem/Integrations/Gem/Gem.yml:45]:
```suggestion
    name: insecure
```

File: Packs/Gem/Integrations/Gem/Gem.py [hunk 2]
```diff
@@ -573,7 +573,7 @@ def init_client(params: dict) -> GemClient:
     """
     return GemClient(
         base_url=params['api_endpoint'],
-        verify=True,
+        verify=not params.get('unverify', False),
```
Comment (RotemAmit) [inline Packs/Gem/Integrations/Gem/Gem.py:576]:
```suggestion
        verify=not params.get('insecure', False),
```

File: Packs/Gem/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Gem
+
+- Added "Trust any certificate" instance param.
```
Comment (RotemAmit) [inline Packs/Gem/ReleaseNotes/1_0_1.md:6]:
```suggestion
- Added the "Trust any certificate" parameter to the integration configuration. 
```

**Review by RotemAmit — COMMENTED** [review]
Thank you very much for your contribution! 
Nice work!

**Review by RotemAmit — APPROVED** [review]
Thank you very much for your contribution! Great work!

**Pull request merged**